ocean_rough.F90

```diff
@@ -228,6 +230,43 @@ subroutine compute_ocean_roughness ( ocean, u_star,  &
           rough_moist = 0.0
       endwhere
 
+   else if (trim(rough_scheme) == 'hwrf17') then
+
+      rough_mom_init = 1.e-03
+      ustar_min = 1.e-05
```

Comment:
I assume this is legacy (as in coare3.5), do we hit the minimum ever/often?

Reply:
Yes, this was borrowed from the part in COARE3.5. I do not think it hits the minimum often.